test/cli/file-sigils-json/file-sigils-json.out

```diff
@@ -1,9 +1,2784 @@
 ----- ./*.rb -----------------------------------------------------------
 No errors! Great job.
-[{"file":"./compiled-false.rb","compiled":"false"},{"file":"./compiled-none.rb","compiled":"none"},{"file":"./compiled-true.rb","compiled":"true"}]
+{
+ "files": [
+  {
+   "path": "https://github.com/sorbet/sorbet/tree/master/bazel-out/host/bin/rbi/procs.rbi",
+   "min_error_level": "Max"
```

Comment:
it's not clear to me why sigil and strict are missing here.

also, compiled is always missing.

Reply:
actually, it looks like zero values might be elided?

Reply:
Was including the rbis intentional? They look like new additions in this PR.

Reply:
> actually, it looks like zero values might be elided?

I think that must be it? `compiled-false.rb` and `compiled-true.rb` have it.

Reply:
> Was including the rbis intentional? They look like new additions in this PR.

They are included because they are in the symbol table.

Reply:
Fair enough, do you know why they were missing before?

Reply:
Resolved in a DM: the `file-sigils-json` printer was omitting them explicitly.